tests/jest/unit/app/modules/import-export/assets/js/admin.test.js

```diff
@@ -0,0 +1,121 @@
+describe( 'Import Export Admin Test', () => {
+	window.elementorAppConfig = {
+		'import-export': {
+			lastImportedSession: {
+				kit_title: 'Kit Title',
+			},
+		},
+	};
+
+	require( 'elementor/app/modules/import-export/assets/js/admin.js' );
+
+	it( 'should check if "kit deleted" dialog is not shown without cache', () => {
+		// Arrange
+		window.elementorCommon = {
+			dialogsManager: {
+				createWidget: jest.fn( () => {
+					return {
+						show: jest.fn( () => {} ),
+					};
+				} ),
+			},
+		};
+
+		// Act
+		window.dispatchEvent( new Event( 'load' ) );
+
+		// No Kit referrer dialog shown without refferer in cache
+		expect( window.elementorCommon.dialogsManager.createWidget ).not.toHaveBeenCalled();
+	} );
+
+	it( 'should check if the referred "kit deleted" dialog is shown', () => {
```

Comment:
when it should be shown?
A nice convention I found goes something like this:
`it( should check if ??? is ??? when ??? )`
`it( should do ??? when ??? )`

Maybe you can try using it in your tests